Source/Core/DolphinQt2/Config/Graphics/GeneralWidget.cpp

```diff
@@ -0,0 +1,332 @@
+// Copyright 2017 Dolphin Emulator Project5~5~5~
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Config/Graphics/GeneralWidget.h"
+
+#include <QCheckBox>
+#include <QComboBox>
+#include <QFormLayout>
+#include <QGridLayout>
+#include <QGroupBox>
+#include <QHBoxLayout>
+#include <QMessageBox>
+#include <QVBoxLayout>
+
+#include "Core/Config/GraphicsSettings.h"
+#include "Core/ConfigManager.h"
+#include "DolphinQt2/Config/Graphics/GraphicsBool.h"
+#include "DolphinQt2/Config/Graphics/GraphicsChoice.h"
+#include "DolphinQt2/Config/Graphics/GraphicsWindow.h"
+#include "UICommon/VideoUtils.h"
+#include "VideoCommon/VideoBackendBase.h"
+#include "VideoCommon/VideoConfig.h"
+
+GeneralWidget::GeneralWidget(X11Utils::XRRConfiguration* xrr_config, GraphicsWindow* parent)
+    : GraphicsWidget(parent), m_xrr_config(xrr_config)
+{
+  CreateWidgets();
+  LoadSettings();
+  ConnectWidgets();
+  AddDescriptions();
+  emit BackendChanged(QString::fromStdString(SConfig::GetInstance().m_strVideoBackend));
+
+  connect(parent, &GraphicsWindow::BackendChanged, this, &GeneralWidget::OnBackendChanged);
+  connect(parent, &GraphicsWindow::EmulationStarted, [this] { OnEmulationStateChanged(true); });
+  connect(parent, &GraphicsWindow::EmulationStopped, [this] { OnEmulationStateChanged(false); });
+}
+
+void GeneralWidget::CreateWidgets()
+{
+  auto* main_layout = new QVBoxLayout;
+
+  // Video Section
+  auto* m_video_box = new QGroupBox(tr("Video"));
+  m_video_layout = new QFormLayout();
```

Comment:
Using a `QFormLayout` here makes this look weird on macOS (and maybe KDE), and the non-form row at the bottom (V-Sync and Use Fullscreen) make it not be fully HIG compliant (it should be centered horizontally on macOS, looks neither macOS-style nor cross-platform-style as-is):

<img width="489" alt="screen shot 2017-06-27 at 12 59 05 pm" src="https://user-images.githubusercontent.com/594093/27607351-74bdfdf8-5b38-11e7-9d71-da159b7582cb.png">

Can you use a simple `QGridLayout`?